homeassistant/components/plugwise/coordinator.py

```diff
@@ -29,15 +33,15 @@ class PlugwiseData(NamedTuple):
 class PlugwiseDataUpdateCoordinator(DataUpdateCoordinator[PlugwiseData]):
     """Class to manage fetching Plugwise data from single endpoint."""
 
-    def __init__(self, hass: HomeAssistant, api: Smile) -> None:
+    _connected: bool = False
+
+    def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
         """Initialize the coordinator."""
         super().__init__(
             hass,
             LOGGER,
-            name=api.smile_name or DOMAIN,
-            update_interval=DEFAULT_SCAN_INTERVAL.get(
-                str(api.smile_type), timedelta(seconds=60)
-            ),
+            name=DOMAIN,
+            update_interval=timedelta(seconds=60),
```

Comment:
This change is not correct, the P1 has an update interval of 10 seconds. 
Please keep the `.get()` construct there, it's needed for the different update_intervals between devices.

Reply:
It has been moved... We can't set it until a connection is made, hence it sets a default and changes it on connect.

Reply:
Sorry, I'd didn't see the fully picture, please ignore the above comment.